services/frontend/src/components/profileForms/formTitle/FormTitle.scss

```diff
@@ -0,0 +1,4 @@
+.prim-gedsInfoLink {
+  display: inline;
+  float: right;
+}
```

Comment:
1. For this file, you should rename it to `FormTitleView.scss`
2. `prim` in the class name probably refers to the primary form, should it be `formTitle` instead?